src/ex4/server/services/item_manager.js

```diff
@@ -1,63 +1,76 @@
-import { readFile, writeToFile } from '../helper/fileHandler.js'
 import { isPokemon, isListOfPokemons, pokemonExistsInTodoList } from '../helper/pokemonHelper.js'
 import { getPokemon, getAllPokemons } from '../clients/pokemon_client.js'
+import items from '../db/models/items.cjs'
+import { Sequelize } from 'sequelize'
 
 class ItemManager {
 
     constructor() {
-        this.filePath = "./server/data/data.json" //Why is this not working??
+        this.sequelize = new Sequelize('todo_db', 'root', 'password', {
+            host: 'localhost',
+            dialect: 'mysql'
+        });
     }
 
     async getItems(req, res) {
         try {
-            const items = await readFile("./server/data/data.json")
-            res.status(200).json(items)
+            let newItems = await items(this.sequelize).findAll()
```

Comment:
since you're not changing the reference of newItems it's better to use `const` instead of `let`.
you also don't need to pass `this.sequalize` here, it is already binded in your index file